cpu/efm32/periph/Kconfig.timer

```
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config EFM32_XTIMER_USE_LETIMER
+    bool "Xtimer uses letimer"
+    depends on CPU_COMMON_EFM32 && USEMODULE_XTIMER
+    default n
+    help
+        Xtimer will use letimer as its low level timer.
```

Do you think it may make sense to add a menu to wrap the platform-specific options in? Mainly to make it more obvious to the user that these options are provided by the platform and are not to be expected for all of them.
```suggestion
menu "EFM32 timer peripheral driver configuration"

config EFM32_XTIMER_USE_LETIMER
    bool "Xtimer uses letimer"
    depends on CPU_COMMON_EFM32 && USEMODULE_XTIMER
    default n
    help
        Xtimer will use letimer as its low level timer.

endmenu
```

---

I think the symbol name indicates this already, I think adding an extra menu just adds more layers and hides the configuration options.

---

In the menuconfig interface the symbols are not shown by default, unless the user toggles the 'show-name' mode or enters the Symbol information. That was my only concern.

---

Maybe I can clarify the help then?

---

"Xtimer will use efm32 letimer as its low level timer"

---

Or "Xtimer will use efm32 low energy timer as its low level timer"

---

Did that